### PR TITLE
1970 - Add uplift-specific sass files for components

### PR DESCRIPTION
**Pull request by clepore**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**
Adds a way to implement files specific to uplift only.

- [x] Verify with @EdwardCoyle that the custom builder doesn't need tweaked to handle these new files.
    - Update: See https://github.com/infor-design/enterprise/issues/1970#issuecomment-487588088

**Related github/jira issue (required)**:
#1970 

**Steps necessary to review your pull request (required)**:
1. `npm run start`
    - verify the styles that were originally in `src/themes/uplift-theme.scss` are still applied in the app (see the diff)

---

🚫 An e2e or functional test for the bug or feature.
🚫 A note to the change log.

### CODE REVIEW

**Comment by clepore** [comment]
Yep, it found an example file ref in the directory that I forgot to remove.

**Comment by EdwardCoyle** [comment]
I left a [comment about this](https://github.com/infor-design/enterprise/issues/1970#issuecomment-487588088) detailing the changes needed for the build.  I'd be happy to help get that up and running.

**Comment by clepore** [comment]
After looking over it, you could probably do it faster than I could, so be my guest, @EdwardCoyle. There may or may not be an -`uplift.scss` for each component. I think that's the only stipulation.

**Comment by EdwardCoyle** [comment]
**Noting:** As of my two commits, running `npm run build` is properly including uplift-specific CSS files in `dist/css/uplift-theme.css`.  Since we know there is a `_header-uplift.scss` source code file, I tested by doing the following:

- run `npm run build` to kick off a standard build
- open `dist/css/uplift-theme.css` after the build completes
- search for `.header h1 {` in the file.  There are 5 entires, but the very last one sits on its own just above the CSS for the hierarchy component, and is coming from the `_header-uplift.scss` source code.
- running the same search on the other three themes should result in only 4 entries, and the one above should not be present in any of those.

Screen Shot 2019-05-03 at 12 21 12 PM

**Another note:** this should work when running a custom build as well, but there are some known issues with some component combinations that will throw errors when trying to build CSS with the custom builder.  Until we fix all open custom builder tickets, we can't expect that it's working 100%.

**Comment by clepore** [comment]
@EdwardCoyle If that is as good we can get it until now, and it doesn't break anything we possibly had before, my vote is to :shipit:

File: scripts/build.js [hunk 1]
```diff
@@ -505,6 +507,8 @@ function sortFilesIntoBuckets(files, srcFilePath) {
   const startOfMid = getFurthestIndexOf(componentsJSFile, searchTerms.mid);
   const startOfComplex = getFurthestIndexOf(componentsJSFile, searchTerms.complex);
 
+  debugger;
```
Comment (tmcconechy) [inline scripts/build.js]: Can be removed

File: scripts/build.js [hunk 2]
```diff
@@ -949,6 +966,8 @@ if (!commandLineArgs.components) {
   requestedComponents = commandLineArgs.components.split(',');
 }
 
+debugger;
```
Comment (tmcconechy) [inline scripts/build.js]: Remove?

**Comment by EdwardCoyle** [comment]
@deep7102 @jmacaluso711 @ericangeles @davidcarlsonberg I'm purposely not reviewing this one since I did a bunch of work for it... aside from Tim, up to you guys to ✅ it over.

**Pull request merged**